tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -3539,3 +3539,928 @@ Feature: an user gets the resources shared to them
+                                  "pattern":"^%user_id_pattern%$"
+                                }
+                              },
+                              "required": [
+                                "displayName",
+                                "id"
+                              ]
+                            }
+                          }
+                        },
+                        "invitation": {
+                          "type": "object",
+                          "required": ["invitedBy"],
+                          "properties": {
+                            "invitedBy": {
+                              "type": "object",
+                              "required": ["user"],
+                              "properties": {
+                                "user": {
+                                  "type": "object",
+                                  "required": [
+                                    "displayName",
+                                    "id"
+                                  ],
+                                  "properties": {
+                                    "displayName": {
+                                      "type": "string",
+                                      "enum": ["Alice Hansen"]
+                                    },
+                                    "id": {
+                                      "type": "string",
+                                      "pattern": "^%user_id_pattern%$"
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        },
+                        "roles": {
+                          "type": "array",
+                          "maxItems": 1,
+                          "minItems": 1,
+                          "items": {
+                            "type": "string",
+                            "pattern": "^%role_id_pattern%$"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+
+
+  Scenario: user lists the folder shared with them from project space after the sharer is deleted
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has created a folder "folder" in space "new-space"
+    And user "Alice" has sent the following share invitation:
+      | resource        | folder    |
+      | space           | new-space |
+      | sharee          | Brian     |
+      | shareType       | user      |
+      | permissionsRole | Viewer    |
+    And the administrator has deleted user "Alice" using the provisioning API
```

Comment:
```suggestion
    And the administrator has deleted user "Alice" using the provisioning API
```
Aren't we supposed to use Graph API instead of provisioning API?

Reply:
I think, it's just the naming there. should use graph